src/test/java/com/exasol/utils/JsonHelperTest.java

```diff
@@ -55,7 +55,7 @@ void testGetKeyAsString() {
     }
 
     @Test
-    void testPrettyJson() {
-        assertThat(JsonHelper.prettyJson(this.expectedJsonObject), equalTo(JSON));
+    void testPrettyJson() throws JSONException {
+        JSONAssert.assertEquals(JSON, JsonHelper.prettyJson(this.expectedJsonObject), false);
```

Comment:
Check argument order. If I remember correctly it is the opposite of want we are used too in case of JSONAssert.

Reply:
fixed